Makefile

```diff
@@ -12,3 +12,4 @@ release:
 clean:
 	$(MAKE) -f Makefile.psp clean
 	$(MAKE) -f Makefile.tools clean
+	rm -rf **/*.PBP **/*.SFO
```

Comment:
This should not really be needed, since the other Makefiles should take care of it.

Reply:
I created it, because the files were still there after a `make clean` not really sure who is the "guilty one"